ceph-dev-setup/build/build

```diff
@@ -104,38 +124,18 @@ else
 
     if [ -x make-dist ]
     then
-      ./make-dist $cephver
-      vers=`ls ceph-*.tar.bz2 | cut -c 6- | sed 's/.tar.bz2//'`
-      extension="tar.bz2"
-      extract_flags="jxf"
-      compress_flags="jcf"
+        ./make-dist $cephver
+        vers=`ls ceph-*.tar.bz2 | cut -c 6- | sed 's/.tar.bz2//'`
+        extension="tar.bz2"
+        extract_flags="jxf"
+        compress_flags="jcf"
     else
-      echo "Running autogen.sh ..."
-      ./autogen.sh
-      echo "Running configure ..."
-      ./configure \
-        --disable-option-checking \
-        '--prefix=/usr' \
-        '--sbindir=/sbin' \
-        '--localstatedir=/var' \
-        '--sysconfdir=/etc' \
-        '--with-debug' \
-        '--with-nss' \
-        '--with-radosgw' \
-        '--disable-static' \
-        '--without-lttng' \
-        $FLAVOR_FLAGS \
-        'CFLAGS= -Wno-unused-parameter' \
-        'CXXFLAGS=  -Wno-unused-parameter' \
-        --cache-file=/dev/null \
-        --srcdir=.
-
-      make dist
-      make dist-bzip2
-      vers=`ls ceph-*.tar.gz | cut -c 6- | sed 's/.tar.gz//'`
-      extension="tar.gz"
-      extract_flags="zxf"
-      compress_flags="zcf"
+        make dist
+        make dist-bzip2
```

Comment:
is there any reason we want to do the "make dist" twice? maybe we can just always release the .bz2 tarball. and share the following code in both branches (cmake and autotools)
```sh
  vers=`ls ceph-*.tar.bz2 | cut -c 6- | sed 's/.tar.bz2//'`
  extension="tar.bz2"
  extract_flags="jxf"
```

and maybe we can dispense with `extension` and `extract_flags`, as they are fixed.
and we don't use  `compress_flags` anywhere, so it can be dropped?